docs/fast_data/configuration/projection_storer.md

```diff
@@ -0,0 +1,1048 @@
+---
+id: projection_storer_config
+title: Projection Storer Configuration
+sidebar_label: Projection Storer
+---
+
+Projection Storer is the service in charge of constantly keeping up-to-date projections records with respect to change events
+that are received from the associated System of Records. Once modifications are stored to the database, the service
+triggers downstream components with the proper mechanism, so that Single Views can be regenerated with latest data. 
```

Comment:
```suggestion
Projection Storer is the service in charge of keeping up-to-date projections records with respect to change events
received from the associated CDC. Once modifications are stored to the database, the service
triggers downstream components with the proper mechanism, so that Single Views can be regenerated with latest data. 
```